README.md

```diff
@@ -12,13 +12,16 @@ This is the server component of the AeroGear metrics service. It is a RESTful AP
 
 ## Clone and Install Dependencies
 
-1. Clone this repository
+1. Clone this repository to `$GOPATH/src/github.com/aerogear/aerogear-metrics-api`
+```sh
+mkdir -p $GOPATH/src/github.com/aerogear
+cd $GOPATH/src/github.com/aerogear
+gir clone git@github.com:aerogear/aerogear-metrics-api.git
 ```
-go get github.com/aerogear/aerogear-metrics-api
-```
-1. Run the following command to build binary
+
+2. Run the following command install dependencies
 ```
-make build
+dep ensure
```

Comment:
-1.. That's included in makefile

Reply:
Any improvements should be done in makefile

Reply:
There's a bunch of places where this is called directly instead of through make, i.e. CI config, dockerfile. Could change those too if possible to run make.